.github/how_to_compile_gudhi_in_a_conda_env.md

```diff
@@ -36,17 +56,16 @@ cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$CONDA_PREFIX -DWITH_GUDHI_
 
 ### Specific python compilation
 
-In order to compile only python module
+In order to compile only the python module, it is the same process, but just change to `src/python` directory:
 ```bash
 cd /workdir/gudhi-devel
 rm -rf build; mkdir build  # /!\ any existing build folder will be removed
 cd build
 cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=$CONDA_PREFIX ..
 cd src/python
-# To build python module in parallel
-python setup.py build_ext -j 16 --inplace  # 16 is the number of CPU that are used to compile the python module. Can be any other value.
+make -j 16  # 16 is the number of CPU that are used to compile the python module. Can be any other value.
 # to clean the build
-# python setup.py clean --all
+# make clean
 ```
 
 In order to use freshly compiled gudhi python module:
```

Comment:
```suggestion
In order to use the gudhi python module you just compiled as explained above:
```

Is there also a way to directly install the compiled version into their conda environment with pip? (something like `pip install --no-build-isolation .`, I am not sure how the CMake/toml position plays into this)